hoodie-common/src/main/java/com/uber/hoodie/common/model/CompactionOperation.java

```diff
@@ -83,7 +80,24 @@ public String getPartitionPath() {
     return partitionPath;
   }
 
-  public Map<String, Object> getMetrics() {
+  public Map<String, Long> getMetrics() {
     return metrics;
   }
+
+  /**
+   * Convert Avro generated Compaction operation to POJO for Spark RDD operation
+   * @param operation Hoodie Compaction Operation
+   * @return
+   */
+  public static CompactionOperation fromHoodieCompactionOperation(HoodieCompactionOperation operation) {
```

Comment:
rename this, something like convertFromAvro.. or something similar to denote what it's doing ?

Reply:
done